config/config.dist.php

```diff
@@ -141,6 +141,7 @@
 $conf['settings']['recaptcha']['enabled'] = 'false';
 $conf['settings']['recaptcha']['public.key'] = '';
 $conf['settings']['recaptcha']['private.key'] = '';
+$conf['settings']['recaptcha']['request.method'] = 'curl'; // options are curl, post or socket. default: post
```

Comment:
This basically makes the default curl. Why not make it `post`?

Reply:
Post uses `file_get_contents` which requires the option `allow_url_fopen` to be enabled. Because this could [cause security issues](https://stackoverflow.com/questions/24049534/is-allow-url-fopen-safe) and therefore it might not be enabled by default. On the other side I can hardly imagine a PHP installation without CURL.

Reply:
I made post the default anyway so, that existing installations have the same behavior as before, when the setting is not added.